include/glow/ExecutionContext/TraceEvents.h

```diff
@@ -129,12 +129,14 @@ struct TraceInfo {
 };
 
 /// The amount and type of TraceEvents that should appear in the trace.
-enum class TraceLevel {
-  NONE,     // No trace events.
-  RUNTIME,  // Glow runtime events only.
-  OPERATOR, // Backend operator instrumentation only.
-  STANDARD, // Glow runtime events and backend operator events.
-  DEBUG     // Full debug events with extra information.
+enum TraceLevel {
```

Comment:
I'm guessing the motivation for converting this to a regular enum was to enable bitmasking behaviour but I *think* a consequence of this is loss of independent scope.

I wouldn't be worried if this was in a class, but this is out in the open (well, inside `namespace glow`). Can we see someone trying to define `NONE`, DEBUG`, or `STANDARD` somewhere else later?

Reply:
Good call, we should make it a member of TraceEvent probably.